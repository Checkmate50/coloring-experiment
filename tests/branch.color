x : int;
y : float;

x < y;

if {
    int;
}
else {
    int;
}
float;