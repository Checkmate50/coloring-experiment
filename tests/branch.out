if {
    x;
}
else {
    x;
}
y;